config/locales/server.en.yml

```diff
@@ -300,3 +300,8 @@ en:
         Images are the same, but prefixed with an exclamation mark.
 
         ![loomio logo](https://i.imgur.com/Fvtworb.png)
+
+  beta_banner:
+    new_loomio_html: |
+      <strong>The New Loomio is here!</strong> It's faster, prettier, and mobile-ready. More info...
```

Comment:
Blog post is ready when you are.

On Tue, Apr 12, 2016 at 10:00 AM James Kiesel <notifications@github.com>
wrote:

> In config/locales/server.en.yml
> <https://github.com/loomio/loomio/pull/3182#discussion_r59289308>:
>
> > @@ -300,3 +300,8 @@ en:
> >          Images are the same, but prefixed with an exclamation mark.
> >
> >          ![loomio logo](https://i.imgur.com/Fvtworb.png)
> > +
> > +  beta_banner:
> > +    new_loomio_html: |
> > +      <strong>The New Loomio is here!</strong> It's faster, prettier, and mobile-ready. More info...
>
> How is our blog post coming along @rdbartlett
> <https://github.com/rdbartlett> ? We should get this thing, plus the
> angular modal thing built and merged.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3182/files/64a90e8b7207e6b82cd6f251461f9866ac2e39ee#r59289308>
>